portfolio/src/main/webapp/slideshow.js

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {removeClassName} from "./tab.js";
+
+/** 
+ * Global variables that gives me which 
+ * slide I'm currently on in the SlideShow and which one I was in previously.
+ */
+let slideIndex = 1;
+let savedIndex = 0;
+
+/** 
+ * Changes the current slide in a slideshow 
+ * by incrementing by the given n. 
+ * @param {number} slideIncrement - The increment/decrement value for 
+ *   showing the next slide.
+ */
+const showNextSlide = (slideIncrement) => {
+  showSlide(slideIndex += slideIncrement);
+}
```

Comment:
Will you ever want to go forward/backward by more than 1? Consider changing the param to a boolean to indicate the direction.

Reply:
They way the slideshow works, the "slideIncrement" can be negative, thus indicating which direction I'm going. The arrow buttons don't allow the user to go by increment/decrement more than one. Clicking the arrows in the slideshow in rapid succession doesn't change anything either, the slideshow still works as intended. And if I want to go more than one 1, I can click on the images in the thumbnail selector and slideIndex changes appropriately.